.huskyrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "hooks": {
```

Comment:
just to maintain unified, you can use package.json config
https://github.com/Wechoo/readit/blob/master/package.json#L37

or remove from package an include all our configs from husky here